inst/schema/spec.md

```diff
@@ -0,0 +1,229 @@
+This is all highly uncertain and liable to change!
+
+# Data
+## POST /data/tab-name/
```

Comment:
Not sure whether tab based endpoints are correct - possible both tabs use the same data?